README.md

```diff
@@ -75,7 +75,7 @@ Use these environment variables to change the default behaviour of the container
 |`-e DECONZ_VNC_PORT=5900`|Default port for VNC mode is 5900; this option can be used to change this port|
 |`-e DECONZ_VNC_PASSWORD=changeme`|Default password for VNC mode is 'changeme'; this option can (should) be used to change the default password|
 |`-e DECONZ_VNC_PASSWORD_FILE=/var/run/secret`|Per default this is disabled and DECONZ_VNC_PASSWORD is used. More details on secrets can be found in the [corresponding section from the official documentation](https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets) |
-|`-e DECONZ_NOVNC_PORT=6080`|Default port for NOVNC is 6080; this option can be used to change this port|
+|`-e DECONZ_NOVNC_PORT=6080`|Default port for NOVNC is 6080; this option can be used to change this port; setting the port to `0` will disable the noVNC functionality|
```

Comment:
Here you use two different capitalizations for `noVNC`.

There is another location in the `README` where the `switch` should be considered:
>By enabling VNC you also enabled noVNC which allows you to connect using a browser. Per default the port is been set to 6080 and if your are not using "--host" networking you need to open the port using the -p directive. Access is through https://hostname:6080/vnc.html, this is a self signed SSL certificate so you need to accept it before you can access the page.

Reply:
What would be the motivation to place it there?

Reply:
`By enabling VNC you also enabled noVNC` suggests that `VNC` and `noVNC` come as twins.
If you look closer at the options you can see that disabling is possible. I thought this should be mentioned here. If not - I'm fine with it ;-)